scripts/Muon_Analysis.py

```diff
@@ -8,9 +8,10 @@
 from __future__ import (absolute_import, division, print_function)
 from Muon.GUI.MuonAnalysis.muon_analysis_2 import MuonAnalysisGui
 from qtpy import QtCore
+from Muon.GUI.Common.usage_report import register_GUI
 
 
-Name = "Muon_Analysis_2"
+Name = "Muon_Analysis"
```

Comment:
Does this mean that the usage reports will be mixed with the old UI from previous versions?

We want the GUI menu items to take over the name, but internally for usage reports, we might want to keep them separate